File: C/EX37.cpp
#include <conio.h>
#include <stdlib.h>
#include <stdio.h>
#include <time.h> 
#include <string.h>
#include <math.h>
#include <locale.h>
#include <ctype.h>

int num[20];
int quant,a,b,consult;

int main()
{

ab:
printf("insira  quantos numeros quer armazenar:\n"); 
scanf("%d", &quant);
if(quant>20)
{
	printf("Insira um valor menor que 20\n");
	goto ab;
}

printf("insira  os numeros desejados:\n");
  
  
for(a=0; a<quant; a++)
  {
      scanf("%d", &num[a]);
  }


 
printf("insira  insira um valor para consultar:\n");  
scanf("%d", &consult);
 
for(b=0; b<quant ; b++)
  {
      
      if(consult==num[b])
      {
      	printf("A posicao deste numero e: %d \n",b);
	  }
  }
  
}
           

File: Python/EX37.py
num = []
x=21
z=0
while(x>20):
    x = int(input('insira  quantos numeros quer armazenar:'))
    if(x>20):
        print('Insira um valor menor que 20')
        

print('insira  os numeros desejados')
   
 
for i in range(0, x, 1):
    b = int(input())
    num.append(b)

c = int(input('insira um valor para consultar:'))

for i in range(0, x, 1):
    if(c==num[i]):
        print('A posicao deste numero e:',i)
        z=1 
if(z!=1):
   print('Este numero não existe no vetor') 
	  
  
  
